src/get_manifest.ts

```diff
@@ -0,0 +1,131 @@
+import { deepMerge, getProtocolInterface, path } from "./deps.ts";
+import {
+  forEachValidatedManifestFunction,
+  getDefaultExport,
+} from "./utilities.ts";
+
+// Responsible for taking a working directory, and an output directory
+// and placing a manifest.json in the root of the output directory
+
+/**
+ * Returns a merged manifest object from expected files used to represent an application manifest:
+ * `manifest.json`, `manifest.ts` and `manifest.js`. If both a `json` and `ts`/`js` are present, then
+ * these `ts`/`js` export will be merged over the `json` file.
```

Comment:
> If both a `json` and `ts`/`js` are present, then these `ts`/`js` export will be merged over the `json` file.

Is this added comment true with the new logic? The way I remember it working was that we start with the JSON file, then check if a TS file exists. If a TS file exists we layer that on top and don't merge the JS file.